CleverPush/Source/CPBannerCardContainer.m

```diff
@@ -44,7 +44,11 @@ - (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(N
 }
 
 - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
-    return [self.blocks count];
+    if (self.blocks.count == 0 || self.blocks == nil ) {
```

Comment:
calling `.count` on a `nil` variable will crash. We need to switch the conditions and check for nil first before accessing .count. Also code style (too much space before ) )